### PR TITLE
UI(issue#6234) : Manage tab removal

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

1. I worked on the worked on removing the manage tabs for -

- Entity Details
- Service Details
- Database Details
- Database Schema Details

2. Added functionality to edit owner and deleting services, databases and database schemas.
3. Added functionality to edit tier in mlModel entities.
4. Made changes in the unit tests and utils for above changes.
5. Fixed the failing cypress tests.

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :
<p align="center">
Screenshot 2022-08-04 at 11 19 37 AM
Screenshot 2022-08-04 at 5 44 11 PM
Screenshot 2022-08-04 at 6 12 21 PM
Screenshot 2022-08-04 at 6 12 31 PM
Screenshot 2022-08-04 at 6 12 46 PM

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx [hunk 1]
```diff
@@ -686,42 +665,6 @@ const ServicePage: FunctionComponent = () => {
...
-          service || 'Service',
-          `${pluralize(instanceCount, 'Database')}, ${pluralize(
-            schemaCount,
-            'Schema'
-          )} and ${pluralize(tableCount, 'Table')}`
-        );
-
-      case ServiceCategory.MESSAGING_SERVICES:
-        return getEntityDeleteMessage(
-          service || 'Service',
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx:704]: do we need to remove this function? cc @Sachin-chaurasiya i think this is used to show how many database schema is available in database

**Review by ShaileshParmar11 — COMMENTED** [review]
left a few comments!

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js [hunk 2]
```diff
@@ -186,22 +188,102 @@ export const deleteCreatedService = (typeOfService, service_Name) => {
...
     .should('exist')
     .should('be.visible')
     .should('have.text', `${typeOfService} Service deleted successfully!`);
+
+  //Checking if the service got deleted successfully
+  cy.clickOnLogo();
+
+  cy.wait(1000);
+  cy.get(
+    '.tw-ml-5 > [data-testid="dropdown-item"] > div > [data-testid="menu-button"]'
```
[the remark this answers is not in the document]
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/cypress/common/common.js]: Done

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js [hunk 3]
```diff
@@ -186,22 +188,102 @@ export const deleteCreatedService = (typeOfService, service_Name) => {
...
+  cy.contains('[data-testid="tab"]', `${typeOfService} Service`).click();
+
+  cy.get(`[data-testid="service-name-${service_Name}"]`).should('not.exist');
+};
+
+export const editOwnerforCreatedService = (typeOfService, service_Name) => {
+  cy.goToHomePage();
+
+  cy.get(
+    '.tw-ml-5 > [data-testid="dropdown-item"] > div > [data-testid="menu-button"]'
```
[the remark this answers is not in the document]
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/cypress/common/common.js]: Done

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js [hunk 4]
```diff
@@ -186,22 +188,102 @@ export const deleteCreatedService = (typeOfService, service_Name) => {
...
+  cy.get('[data-testid="list-item"]')
+    .should('exist')
+    .should('be.visible')
+    .click();
+  cy.wait(1000);
+
+  cy.get('[data-testid*="owner"]')
+    .invoke('text')
+    .then((text) => {
+      expect(text).equal('Cloud_Infra');
```
[the remark this answers is not in the document]
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/cypress/common/common.js]: Done

File: openmetadata-ui/src/main/resources/ui/cypress/integration/Pages/EntityDetails.spec.js [hunk 5]
```diff
@@ -29,36 +29,60 @@ describe('Entity Details Page', () => {
...
+      .click();
+
+    cy.wait(1000);
     // check for delete section and delete button is available or not
-    cy.get('[data-testid="danger-zone"]').scrollIntoView().should('be.visible');
-    cy.get('[data-testid="hard-delete"] > .tw-flex')
-      .scrollIntoView()
-      .find('[data-testid="delete-button"]')
+    // cy.get('[data-testid="danger-zone"]').scrollIntoView().should('be.visible');
+    cy.get('[data-testid="delete-button"] > .tw-font-medium')
```
[the remark this answers is not in the document]
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/cypress/integration/Pages/EntityDetails.spec.js]: Done

File: openmetadata-ui/src/main/resources/ui/cypress/integration/Pages/EntityDetails.spec.js [hunk 6]
```diff
@@ -29,36 +29,60 @@ describe('Entity Details Page', () => {
...
       .should('be.visible')
       .click()
       .as('deleteBtn');
 
+    cy.wait(1000);
+
+    cy.get(':nth-child(2) > :nth-child(2) > .tw-text-sm')
```
[the remark this answers is not in the document]
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/cypress/integration/Pages/EntityDetails.spec.js]: Done

File: openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx:704]: This function is added to utils. Now this message will display when one tries to permanently delete a service.

**Comment by ShaileshParmar11** [comment]
@aniketkatkar97 can you resolve the conflict and check for sonarCloud failure?

**Comment by aniketkatkar97** [comment]
> @aniketkatkar97 can you resolve the conflict and check for sonarCloud failure?

Yes, working on it

**Review by ShaileshParmar11 — APPROVED** [review]
LGTM, Great work 🚀

**Pull request merged**